packages/amplify-util-mock/src/__tests__/velocity/model-auth.test.ts

```diff
@@ -777,82 +777,11 @@ describe("@model @primaryIndex @index auth", () => {
       updatedAt: AWSDateTime
     }`;
 
-    const out = transformer.transform(validSchema);
```

Comment:
is this change supposed to be part of the package update PR?

Reply:
Yeah, this was a test that Danielle had updated in the api category repo that needed to be reflected back here as well, so I just cherry-picked that file update in from her initial PR.

Reply:
thanks, LGTM, there is a conflict though